packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -301,8 +301,11 @@ class SemanticsData extends Diagnosticable {
   ///
   /// If this value is non-null, the SemanticsNode must not have any children
   /// as those would be replaced by the semantics nodes of the referenced
-  /// platform view.
-  ///
+  /// platform view, and other semantic configurations are ignored. The ignored
+  /// configurations include: [flags], [actions], [label], [value], [increasedValue],
```

Comment:
This explicit list seems hard to maintain. When we add a new property nobody will remember to add it here.